tests/unmount.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { defineComponent } from 'vue'
+
+import { mount } from '../src'
+
+const AXIOM = 'Rem is the best girl'
+
+describe('Unmount', () => {
+  it('works on single root component', () => {
+    const errorHandler = jest.fn()
+    const Component = {
+      template: `
+        <div>${AXIOM}</div>
+      `
+    }
+    const wrapper = mount(Component, {
+      props: {},
+      global: {
+        config: {
+          errorHandler
+        }
+      }
+    } as any) // The type checking keeps complaning about unmatched type which might be a bug
```

Comment:
Here I tried it with 
```ts
mount(Component, {
  global: {
    config: ...
  }
}
```
Type system seems not agree with this syntax when custom component is mounting with global options

Reply:
Interesting, will deal with the types later, it's outside the scope of this ticket. Great job!